LoRDeckCodes_py3/lor_deck_encoder.py

```diff
@@ -0,0 +1,243 @@
+from base64 import b32encode, b32decode
+
+from card_code_and_count import CardCodeAndCount
+import varint_translator
+
+CARD_CODE_LENGTH = 7
+faction_to_int = {"DE": 0, "FR": 1, "IO": 2, "NX": 3, "PZ": 4, "SI": 5}
+int_to_faction = {0: "DE", 1: "FR", 2: "IO", 3: "NX", 4: "PZ", 5: "SI"}
+MAX_KNOWN_VERSION = 1
+
+
+def get_deck_from_code(code):
+    result = []
+
+    try:
+        bytes_ = b32decode(code + ('=' * (56 - len(code))))
+    except:
```

Comment:
can we set an exception type here? https://moserei.de/2017/09/20/python-bare-except.html for why this can matter :)

Reply:
Yeah I was curious about it, but left it as it was since that's about how it was in the [original](https://github.com/RiotGames/LoRDeckCodes/blob/162617c7d927a1b649724dae1307b204c0740063/LoRDeckCodes/LoRDeckEncoder.cs#L42). Would `Exception` be an ok type to set or is there anything more specific in mind?

Reply:
yeah i think `Exception` is reasonable here since we're just re-raising with a slightly more human readable error. might be worth using `traceback.format_exc()` to include the original stack trace in the error message for debugging purposes? not super important either way though, i think.

Reply:
I discovered while testing it that even with the except, the original stack trace still gets output as well. but yeah I'll add the `Exception` to it.